### ISSUE TITLE
Clarify `filter_prop` in `composite(), mention that it's the *minimum* proportion

### ISSUE THREAD

**Issue by brews** [ltrr-arizona-edu/burnr#127]
Clarify `filter_prop` in `composite() documentation. We need to mention that it's the *minimum* proportion, much like the other arguments.

The name of the `filter_prop` function argument is a bit confusing here, given the other argument names. But I don't think we can change the function signature from `filter_prop` to `filter_min_prop` (or the like) without breaking backwards compatibility for new users. So, simply clarifying this in documentation is the best bet.

**Comment by chguiterman** [comment]
Is there a way to get around the compatibility issue with match.args? Can
we keep filter_prop with default = NULL and also add filter_min_prop ?
--
Christopher Guiterman, Ph.D.
Research Associate
Laboratory of Tree-Ring Research
University of Arizona

**Comment by brews** [comment]
@chguiterman Yes, there is a way around this. We would also put in a deprecation warning for people still using the old `filter_prop` argument. 

In the fix I'm working on (a0453fc91f1c1e60773174500e8c6af838b7f4f1) I opted to just clarify the documentation. I don't want to change the actual arg name because we use `filter_prop` as an arg in several other functions. I feel like we'd need to refactor those function argument names too. It would potentially break lots of noobies code for a very small change (IMHO).

(I hope I got at your question...)
